substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1231,6 +1263,27 @@ impl<T: Config> BondedPool<T> {
 			});
 		};
 	}
+
+	/// Withdraw all the funds that are already unlocked from staking for the
+	/// [`BondedPool::bonded_account`].
+	///
+	/// Also reduces the [`TotalValueLocked`] by the difference of the
+	/// [`T::Staking::total_stake`] of the [`BondedPool::bonded_account`] that might occur by
+	/// [`T::Staking::withdraw_unbonded`].
+	///
+	/// Returns the result of [`T::Staking::withdraw_unbonded`]
+	fn withdraw_from_staking(&self, num_slashing_spans: u32) -> Result<bool, DispatchError> {
+		let bonded_account = self.bonded_account();
```

Comment:
won't ask you to change, but here you are choosing readability over performance: `self.bonded_account` was already known at the call site of this function, and it is non-trivial to re-compute. Wanted to make sure you are aware of that.

Reply:
The way I see it it's only known in one of the two usages.

Reply:
yeah, you are right.